pkg/scheduler/testing/workload_prep.go

```diff
@@ -27,15 +27,15 @@ type keyVal struct {
 	v string
 }
 
-// MakeNodesAndPodsForEvenPodsSpread serves as a testing helper for EvenPodsSpread feature.
```

Comment:
can we avoid renames in what is supposed to be a no-op feature promotion? it makes the changeset harder to reason about

Reply:
Change reverted. I will use another cleanup PR to cover this.